userguide/install.rst

```diff
@@ -115,9 +112,16 @@ the :file:`.iso` file and the device name of the device to write.
    command can be avoided by writing the :file:`.iso` file to a
    CD instead.
 
+
+This example demonstrates writing the image to the first USB device
+(*/dev/da0*) connected to a FreeBSD system. Substitute
+:literal:`FreeNAS-RELEASE.iso` with the filename of the downloaded
+:file:`.iso` file and :literal:`/dev/da0` with the device name of the
```

Comment:
It's probably better to split the sentence here.

...file. Replace :literal:`/dev/da0` with ...